WFCD/WFinfo
# Upgrade tesseract to 4.1.1

**Don't merge this until its tested and ready for immediate release** (on various configs including legacy AVX unsupported CPUs)
Side effect of this upgrade - all users will be required to use latest version of WFInfo (9.3.4?), as old dlls will be deleted from `master` branch upon merge. Ideally we will want to consider moving away these dlls to another location outside of github to avoid such breaking changes.


* Upgrade upstream tesseract to 4.1.1 (refer to [this](https://github.com/tesseract-ocr/tesseract/blob/4.1/ChangeLog) for changelog)
* Upgrade dotnet tesseract wrapper to 4.1.1 (refer to [this](https://github.com/charlesw/tesseract/blob/master/ChangeLog.md) for changelog)
* Update leptonica to 1.80 (required transitive dependency)
* Add cleanup script for old dlls to avoid conflicts

How to test while its in PR stage:
1. Checkout to this branch
2. "Clean WFInfo", "Rebuild WFInfo", launch WFInfo, see if it loads fully without crashes, try to process some valid warframe image (perhaps via debug Shift + Activation key)

PS: The rollback to older version of WFInfo will require manual deletion of tesseract4 folder (or dlls inside of it)
